docker-compose-only/docker-compose-dev.yml

```diff
@@ -0,0 +1,60 @@
+# This is our development docker-compose file, building on top of the prod one.
+# This file adds a synchronisation between ./app and /var/www, that has the same
+# performance characteristics as the default native_osxfs scheme defined in
+# ../default/.
+# This example configuration runs the same docker image as ../default, with the
+# same environment. It does not need a docker-sync daemon running on your host
+# though, so you can bring up your development environment using
+# `docker-compose up -d` without first running `docker-sync start`. If you wish
+# to read the logs of the sync process, use `docker-compose logs app-syncer`.
+#
+# If you have a native_osxfs based docker-sync configuration that you want to
+# turn into a docker-compose only configuration, run `docker-sync start` and
+# then use `docker inspect -f '{{ .Mounts }}' $container_id` and
+# `docker exec $container_id env` to work out the elements needed to create a
+# docker-compose based container definition.
+
+version: "2"
+services:
+  app:
```

Comment:
If you want me to change it to `app-docker-compose-only` to be more consistent with the other examples then I can. Happy to take other name suggestions too.

Reply:
yes this would make sense to avoid conflicts when people try different examples. Thanks!